nginx.conf

```diff
@@ -8,18 +8,41 @@ http {
  
     upstream docker-nginx {
         server web-verification:80;
+        server ping-verification:80;
     }
  
     server {
         listen 80;
  
-        location /web {
+        location / {
             proxy_pass         http://docker-nginx/;
             proxy_redirect     off;
             proxy_set_header   Host $host;
             proxy_set_header   X-Real-IP $remote_addr;
             proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header   X-Forwarded-Host $server_name;
         }
+
+         location /web {
+            rewrite ^/web(.*) / break;
+
+            proxy_pass         http://web-verification:80;
+            proxy_redirect     off;
+            proxy_set_header   Host $host;
+            proxy_set_header   X-Real-IP $remote_addr;
+            proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header   X-Forwarded-Host $server_name;
+        }
+
+        location /icmp {
+            rewrite ^/icmp(.*) / break;
```

Comment:
@MatiasMinchaca why this?

Reply:
was used to rewrite the location but I saw that it is unnecessary